docs/source/execute_api.rst

```diff
@@ -0,0 +1,145 @@
+Executing notebooks
+===================
+
+In this section we show how to execute a ``.ipynb`` notebook
+document saving the result in notebook format.
+To export the notebook to other formats see the
+:ref:`next section <nbconvert_library>`.
```

Comment:
It's probably better to refer to this by name, because if we rearrange the docs or put another page in between, we won't think about this.